### ISSUE TITLE
Insufficient color contrast on active sort button

### ISSUE THREAD

**Issue by nichgray** [CDRH/lewisandclark#241]
This fella on the search page, I think: 

<img width="64" height="68" alt="Image" src="https://github.com/user-attachments/assets/73c25909-5751-4d36-ac9a-d530b22af221" />

...so probably the "Search Only Journals" button, too.
